java-core/google-cloud-core/src/main/java/com/google/cloud/ServiceOptions.java

```diff
@@ -582,6 +596,16 @@ public String getProjectId() {
     return projectId;
   }
 
+  /**
+   * Universe Domain is the domain for Google Cloud Services. It follows the format of
+   * `{ServiceName}.{UniverseDomain}`. For example, speech.googleapis.com would have a Universe
+   * Domain value of `googleapis.com` and cloudasset.test.com would have a Universe Domain of
+   * `test.com`. If this value is not set, this will default to `googleapis.com`.
```

Comment:
> If this value is not set, this will default to `googleapis.com`.

I felt it's unclear what is the second "this" in "this will default to `googleapis.com`". When universe domain is not set, this is null. I think you want to add "when" as in "this will default to `googleapis.com when ...`.

Reply:
I do think the above is the correct behavior. I want to keep consistency with the current GAPICs behavior where the value put in the setter is what is returned by the getter (this is a future enhancement to be able to return the resolved values).

The resolved universeDomain is going to be `googleapis.com` when the inputted universeDomain is null. I'll update the docs to be clearer.